compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/ComposeWindow.uikit.kt

```diff
@@ -315,6 +335,12 @@ internal actual class ComposeWindow : UIViewController {
 
     override fun viewWillLayoutSubviews() {
         super.viewWillLayoutSubviews()
+
+        // UIKit possesses all required info for layout at this point
+        currentInterfaceOrientation?.let {
```

Comment:
> UIKit possesses all required info for layout at this point

Does it mean we can be sure `currentInterfaceOrientation ` is not null?

Can we have `interfaceOrientationState.value = currentInterfaceOrientation!!` ?

Reply:
Yes, but user can still do something stupid like:

```
let vc = ViewController()
vc.view.frame = .init(x: 0.0, y: 0.0, width: 600.0, height: 800.0)
vc.view.layoutIfNeeded()
```

Which will trigger viewWillLayout, and while it doesn't fruit any meaningful outcome (vc is not attached to any UIWindow hierarchy), it will cause a crash.